### ISSUE TITLE
article tags (array) generated as og:tag instead article:tag

### ISSUE THREAD

**Issue by zafranf** [artesaos/seotools#74]
hi, i just try this seo tools and i got this issue when i trying to generate article tag

this is in my controller:

```
OpenGraph::setArticle([
                'published_time' => $par->post_time,
                'author' => $par->author->user_fullname,
                'section' => $category,
                'tag' => $tags
            ]);
```

when the $tags is string, it generated as:
`<meta property="article:tag" content="tag" />`

but when it's an array, it generated as:
`<meta property="og:tag" content="tag" />`

**Comment by mahsa-jannaty** [comment]
I have this problem too! any body has fixed this issue?

**Comment by andredaniel** [comment]
I need to set the property to `restaurant:restaurant` but instead it, I've got `og:restaurant`. Any news about how to fix it?

**Comment by BulgarianHealer** [comment]
@mahsa-jannaty 
@andredaniel 
I had the same problem, but i was able to fix it for now.

I made some changes to OpenGraph class, on eachProperties function after the 1st check for array
Original: 
```php
                $subListPrefix = (is_string($property)) ? $property : $prefix;
                $subList = $this->eachProperties($value, $subListPrefix);

```
Change to:
```php
                if (is_string($property)){
                    $subListPrefix = $prefix.":".$property;
                    $subList = $this->eachProperties($value, $subListPrefix, false);
                } else {
                    $subListPrefix = (is_string($property)) ? $property : $prefix;
                    $subList = $this->eachProperties($value, $subListPrefix);
                }
```

I will think if there is another way to fix this problem, but for now, this is okay.

**Comment by vinicius73** [comment]
Good @BulgarianHealer 
PR is very welcome!

**Comment by BulgarianHealer** [comment]
@vinicius73 Can you check the RP?  It's almost the 2-nd month without merging.

**Comment by vinicius73** [comment]
Thanks @BulgarianHealer 
Can you provide a litle test for your PR?

**Comment by vinicius73** [comment]
Closed by #233
